src/main/java/se/sundsvall/messagingsettings/api/MessagingSettingsResource.java

```diff
@@ -0,0 +1,52 @@
+package se.sundsvall.messagingsettings.api;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+import static org.springframework.http.MediaType.APPLICATION_PROBLEM_JSON_VALUE;
+import static org.springframework.http.ResponseEntity.ok;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.ConstraintViolation;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+import org.zalando.problem.Problem;
+import se.sundsvall.dept44.common.validators.annotation.ValidMunicipalityId;
+import se.sundsvall.messagingsettings.api.model.SenderInfoResponse;
+import se.sundsvall.messagingsettings.service.MessagingSettingsService;
+
+@Tag(name = "Messaging Settings")
+@RestController
+@Validated
+public class MessagingSettingsResource {
+
+	public static final String GET_SENDER_INFO_PATH = "/{municipalityId}/{departmentId}/sender-info";
+
+	private final MessagingSettingsService messagingSettingsService;
+
+	public MessagingSettingsResource(MessagingSettingsService messagingSettingsService) {
+		this.messagingSettingsService = messagingSettingsService;
+	}
+
+	@GetMapping(path = GET_SENDER_INFO_PATH, produces = {
+		APPLICATION_JSON_VALUE, APPLICATION_PROBLEM_JSON_VALUE
```

Comment:
This means 200, 400 and 404 can return both of these.

a 200 will never return APPLICATION_PROBLEM_JSON_VALUE and 4xx will never return APPLICATION_JSON_VALUE